src/lib/libraries/decks/en-steps.js

```diff
@@ -1,60 +1,61 @@
 // Intro
-import introMove from './steps/intro-move.en.gif';
-import introSayHello from './steps/intro-say-hello.en.gif';
+import introMove from './steps/intro-1-move.en.gif';
+import introSay from './steps/intro-2-say.en.gif';
+import introGreenFlag from './steps/intro-3-green-flag.en.gif';
 
 // Text to Speech
 import speechAddExtension from './steps/speech-add-extension.en.gif';
 import speechSaySomething from './steps/speech-say-something.en.png';
 import speechSetVoice from './steps/speech-set-voice.en.png';
 import speechMoveAround from './steps/speech-move-around.en.png';
-import speechAddBackdrop from './steps/speech-add-backdrop.gif';
-import speechAddSprite from './steps/speech-add-sprite.gif';
+import speechAddBackdrop from './steps/animate-char-pick-backdrop.LTR.gif';
```

Comment:
It makes sense to reuse gifs that are identical across LTR languages (no text), but the name of the file is a bit jarring - it's odd to see `animate-char` in the middle of `speech`. It might have been better to use a generic name like `pick-backdrop.LTR.gif`, but this shouldn't hold up the PR.